examples/Demo/Shared/Pages/Design/OfficeColorTable.razor

```diff
@@ -0,0 +1,50 @@
+﻿<style>
+    th, td {
+        border: solid gray 1px;
+        padding: 0 5px;
+    }
+</style>
+<div>
+    <table>
+        <thead>
+            <tr>
+                <th>Product</th>
+                <th>AccentBaseColor</th>
+            </tr>
+        </thead>
+        <tbody>
+            @{
+                var allColors = OfficeColorUtilities.AllColors;
+                @foreach (var color in allColors)
+                {
+                    <tr>
+                        <td>
+                            <FluentLabel>@color</FluentLabel>
+                        </td>
+                        <td>
+                            <FluentStack>
+                                <FluentIcon Value="@(new Icons.Filled.Size20.RectangleLandscape())"
+                                            Color="Color.Custom"
+                                            CustomColor="@GetCustomColor(color)" />
+                                <FluentLabel>@GetCustomColor(color)</FluentLabel>
+                            </FluentStack>
+                        </td>
+                    </tr>
+                }
+            }
+        </tbody>
+    </table>
+
+</div>
+
+@code {
```

Comment:
There is a bit magic here. `#036ac4` I used the same value as used for the `GetCustomColor` method for [SiteSettingsPanel.razor.cs](https://github.com/microsoft/fluentui-blazor/blob/dev/examples/Demo/Shared/Components/SiteSettingsPanel.razor.cs).

Gives me a bad code smell.

Reply:
Yeah, it is not the prettiest but  I can live with it.